modules/ocf/manifests/systemd/service.pp

```diff
@@ -3,22 +3,27 @@
   $content = undef,
   $ensure = 'running',
   $enable = true,
+  $suffix = 'service',
 ) {
-  file { "/etc/systemd/system/${title}.service":
+  if $content and $source {
+    fail('You may not supply both content and source parameters to ocf::systemd::service')
+  }
+  elsif $content == undef and $source == undef {
+    fail('You must supply either the content or source parameter to ocf::systemd::service')
+  }
```

Comment:
Nice, this is quite good to have, and honestly we should have more checks like this on any of our custom definitions (like `ocf::repackage` for example).